py/picca/fitter2/xi.py

```diff
@@ -119,9 +121,58 @@ def xi_qso_radiation(r, mu, k, pk_lin, pk_func, tracer1, tracer2, ell_max=None,
     return xi_drp(r, mu, k, pk_lin, pk_func, tracer1, tracer2, ell_max, **pars) + xi_rad
 
 ### Growth factor evolution
-def growth_function_no_de(z, zref = None, **kwargs):
+def growth_factor_no_de(z, zref=None, **kwargs):
     return (1+zref)/(1.+z)
 
+def cache_growth_factor_de(function):
+    cache = {}
+    def wrapper(*args, **kwargs):
+        Om = kwargs['Om']
+        OL = kwargs['OL']
```

Comment:
Apparently `OL` is something in Python: It is in collored in blue.
Would it be possible to put it as `Ol`?
Obviously, this is not important.

Reply:
I can't find anything that would explain why it's blue... and `Ol` looks much less pretty than `OL`. I'll keep looking while you review other things

Reply:
I agree `Ol` dosn't look nice.